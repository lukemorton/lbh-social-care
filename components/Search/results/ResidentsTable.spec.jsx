import { render } from '@testing-library/react';

import ResidenTable from './ResidentsTable';

describe('ResidenTable component', () => {
  const props = {
    records: [
      {
        personId: 'foo',
        firstName: 'Fname',
        lastName: 'Lname',
        addressList: [
          { address: 'foo_address', postcode: 'foo_postcode' },
          { address: 'bar_address', postcode: 'bar_postcode' },
        ],
        dateOfBirth: '1978-02-23T00:00:00.0000000',
      },
    ],
  };
  it('should render properly', () => {
    const { asFragment } = render(<ResidenTable {...props} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
