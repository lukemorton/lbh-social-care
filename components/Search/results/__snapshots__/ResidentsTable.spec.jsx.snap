// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResidenTable component should render properly 1`] = `
<DocumentFragment>
  <table
    class="govuk-table"
  >
    <thead
      class="govuk-table__head"
    >
      <tr
        class="govuk-table__row"
      >
        <th
          class="govuk-table__header"
          scope="col"
        >
          Mosaic ID
        </th>
        <th
          class="govuk-table__header"
          scope="col"
        >
          Client Name
        </th>
        <th
          class="govuk-table__header"
          scope="col"
        >
          Date of birth
        </th>
        <th
          class="govuk-table__header"
          scope="col"
        />
      </tr>
    </thead>
    <tbody
      class="govuk-table__body"
    >
      <tr
        class="govuk-table__row"
      >
        <td
          class="govuk-table__cell"
        >
          foo
        </td>
        <td
          class="govuk-table__cell"
        >
          Fname Lname
        </td>
        <td
          class="govuk-table__cell"
        >
          2/23/1978
        </td>
        <td
          class="govuk-table__cell"
        >
          <a
            class="govuk-link govuk-custom-text-color"
            href="/people/foo"
          >
            View
          </a>
        </td>
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
